grossmont/computer-science-and-cybersecurity-organization/style.css

```diff
@@ -33,4 +36,14 @@ body {
 body {
   padding-top: 15vh;
   padding-bottom: 15vh;
+}
+
+:root{
+  --primary-color: white;
+  --secondary-color: #171717;
+}
+
+.dark-theme{
+  --primary-color: #171717;
+  --secondary-color:white;
```

Comment:
Think you tried to use this theme for the theme.js file, but it wasn't assigned to any html elements when I reviewed it. However I see what you're trying to do here, and you could actually change the css class of the select jQuery element and toggle it between two theme classes, such as .dark-theme and .light-theme.

For example you could say, in theme.js (pseudocode)
```
if(isCurrentlyDarkMode){
$("body").attr('class', 'light-theme');
}else{
$("body").attr('class', 'dark-theme');
}
```